templates/blog/index.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'base.html' %}
+{% block content %}
+  <div class="container">
+    <h1>Home page</h1>
+    <ul>
+      {% for post in post_list %}
+        <li>
+          <a href="{% url 'blog:post' pk=post.pk %}">{{ post.title }}</a>

```

Comment:
The URL tag uses 'blog:post', but it should be 'blog:post-detail' to match the URL pattern defined in `urls.py`.